engine-comparison/dispatcher.sh

```diff
@@ -7,60 +7,84 @@
 
 build_engine() {
   # [[ ! -e ~/$FENGINE_CONFIGS_DIR/$1 ]] && echo "cant do" && exit 1
-
   # All of this code will probably be in a separate script
 
   FENGINE_CONFIG=$1
 
   echo "Building $FENGINE_CONFIG"
-  export ENGINE_DIR=$WORK/fuzz-engines/${FENGINE_CONFIG}
+  FENGINE_NAME=$(basename $FENGINE_CONFIG)
+  export ENGINE_DIR=$WORK/fuzz-engines/${FENGINE_NAME}
```

Comment:
I was thinking that this would be used in `build_benchmark_with`, but I just realized that this just won't work with concurrency